hack/jenkins/upload-to-gcs.sh

```diff
@@ -153,7 +155,7 @@ function print_started() {
 # Use this to test changes to print_started.
 if [[ -n "${TEST_STARTED_JSON:-}" ]]; then
   version=$(find_version)
-  cat <(print_started) | jq .
+  < <(print_started) jq .
```

Comment:
Why did it change and what did it fix:
```
In ./hack/jenkins/upload-to-gcs.sh line 156:
  cat <(print_started) | jq .
      ^--------------^ SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.

For more information:
  https://github.com/koalaman/shellcheck/wiki/SC2002
```
Example:
**Problematic code**:
```
cat file | tr ' ' _ | nl
cat file | while IFS= read -r i; do echo "${i%?}"; done
```
**Correct code**:
```
< file tr ' ' _ | nl  
while IFS= read -r i; do echo "${i%?}"; done < file
```

Reply:
I don't think this change works correctly. `jq . <(print_started)` seems like the correct change.

Reply:
thanks.